lib/fuzion/std/fileio.fz

```diff
@@ -0,0 +1,9 @@
+public fuzion.std.fileio is
```

Comment:
Please add our standard header as in `out.fz` consisting of the GPLv3 header,  the Tokiwa banner, and a comment giving in the first line the qualified feature name folllowed by `--` and short description followed an optional longer description in the following paragraphs.